.goreleaser.yaml

```diff
@@ -0,0 +1,66 @@
+# This is an example .goreleaser.yml file with some sensible defaults.
+# Make sure to check the documentation at https://goreleaser.com
+before:
+  hooks:
+    # You may remove this if you don't use go modules.
+    - go mod tidy
+builds:
+  - main: ./cmd/celestia-appd
+    binary: celestia-appd
+    env:
+      - CGO_ENABLED=0
+      - SDKPath={{ "github.com/cosmos/cosmos-sdk/version" }}
+    goos:
+      - linux
+      - windows
```

Comment:
I'd remove windows here. Unless we also have a CI workflow that runs tests for that as well.

Reply:
we have not formally discussed which binaries we should create, only casually mention it in conversations

as Rootul has mentioned ubuntu is the only thing we have tests for atm, so its likely the only safe one. I'll also second that if we want to support others, we do so in a different PR